provider/designate/designate.go

```diff
@@ -340,14 +340,6 @@ func (p designateProvider) Records(ctx context.Context) ([]*endpoint.Endpoint, e
 	return result, nil
 }
 
-func (p designateProvider) PropertyValuesEqual(name, previous, current string) bool {
-	if name == designateRecordSetID || name == designateZoneID || name == designateOriginalRecords {
```

Comment:
How are those handled?

Reply:
The designate provider does not consume these as provider-specific properties.